srcalc/src/main/java/gov/va/med/srcalc/domain/model/Rule.java

```diff
@@ -217,7 +217,8 @@ private Expression parseSummandExpression(final String summandExpression)
     }
     
     /**
-     * Returns a String in the form of "#a, #b, #c" that will 
+     * Returns a String in the form of "#a, #b, #c" that will communicate to the
+     * user which variables are available for the summand expression.
      */
     @Transient
     public String getExpressionVariablesString()
```

Comment:
It did work, but I had changed some code in the editRule.jsp so that I didn't need it anymore.